packages/myst-config/src/site/validators.ts

```diff
@@ -153,35 +141,48 @@ export function validateSiteConfigKeys(
     );
     if (domains) output.domains = [...new Set(domains)];
   }
-  if (defined(value.twitter)) {
-    output.twitter = validateString(value.twitter, {
-      ...incrementOptions('twitter', opts),
-      regex: /^@?(\w){1,15}$/,
-    });
-  }
-  if (defined(value.logo)) {
-    output.logo = validateString(value.logo, incrementOptions('logo', opts));
-  }
-  if (defined(value.logo_text)) {
-    output.logo_text = validateString(value.logo_text, incrementOptions('logo_text', opts));
-  }
   if (defined(value.favicon)) {
     output.favicon = validateString(value.favicon, incrementOptions('favicon', opts));
   }
-  if (defined(value.analytics)) {
-    output.analytics = validateSiteAnalytics(value.analytics, incrementOptions('analytics', opts));
-  }
-  if (defined(value.design)) {
-    output.design = validateSiteDesign(value.design, incrementOptions('design', opts));
+  if (defined(value.template)) {
+    output.template = validateString(value.template, incrementOptions('template', opts));
   }
   return output;
 }
 
+/**
+ * Validate and return common, non-template attributes of SiteConfig
+ */
 export function validateSiteConfig(input: any, opts: ValidationOptions) {
   const value = validateObjectKeys(input, SITE_CONFIG_KEYS, {
     ...opts,
     returnInvalidPartial: true,
+    // Template options will appear as extra keys here; no need for warnings.
+    suppressWarnings: true,
```

Comment:
Do the key/values also get dropped?

Reply:
Yes, that is why we need the other `getSiteTemplateOptions` function, to get all the non-explicit keys.

Reply:
Sounds good. Didn't pick that up the first time.